tests/by-util/test_dd.rs

```diff
@@ -1394,3 +1396,69 @@ fn test_sync_delayed_reader() {
     assert_eq!(&output.stdout, &expected);
     assert_eq!(&output.stderr, b"0+8 records in\n4+0 records out\n");
 }
+
+/// Test that a seek on an output FIFO results in a read.
+#[test]
+#[cfg(not(windows))]
+fn test_seek_output_fifo() {
+    let ts = TestScenario::new(util_name!());
+    let at = &ts.fixtures;
+    at.mkfifo("fifo");
+
+    // TODO When `dd` is a bit more advanced, we could use the uutils
+    // version of dd here as well.
+    let child = Command::new("dd")
+        .current_dir(&at.subdir)
+        .args([
+            "count=1",
+            "if=/dev/zero",
+            &format!("of={}", at.plus_as_string("fifo")),
+            "status=noxfer",
+        ])
+        .stderr(Stdio::piped())
+        .spawn()
+        .expect("failed to execute child process");
+
+    ts.ucmd()
+        .args(&["count=0", "seek=1", "of=fifo", "status=noxfer"])
+        .succeeds()
+        .stderr_only("0+0 records in\n0+0 records out\n");
+
+    let output = child.wait_with_output().unwrap();
```

Comment:
If pull request #4136 gets merged, that would allow me to make these tests more concise.

Reply:
I think I was wrong when I made this comment. The `child` process comes from `Command`, not `UCommand`, so I don't think any changes can be made here at the moment.

Reply:
You could use `ts.cmd` to make it a `UCommand`, but it's not a high priority.